### ISSUE TITLE
[Nextclade] consider adding back optional input "input-pcr-primers" for nextclade v3 task

### ISSUE THREAD

**Issue by kapsakcj** [theiagen/public_health_bioinformatics#400]
:cool:

### :pushpin: Explain the Request

We removed the `nextclade run --input-pcr-primers` option from the nextclade_v3 WDL task in #349 due to the migration guide stating that the option was removed in v3 of nextclade. Turns out the migration guide docs were a little out of date and the option was re-implemented in v3.1.0 after nextclade users requested it.

We might consider adding it back if requested by the users of our workflows.

### :books: Context

See the PR linked above for more context

**Comment by AndrewLangvt** [comment]
@kapsakcj I completely agree we would pursue this, if requested by users. I appreciate you creating this issue for tracking/documentation purposes. In an effort to cut down on the list of outstanding issues that we are not actively pursuing, I will close this until a lab does request this work, or we identify a need for the feature improvement, internally.
